src/main/java/uk/org/tombolo/importer/londondatastore/AbstractLondonDatastoreImporter.java

```diff
@@ -17,4 +21,9 @@ public Provider getProvider() {
         return PROVIDER;
     }
 
+    @Override
+    protected List<String> getOaDatasourceIds() {
+        return Collections.singletonList(OaImporter.OaType.localAuthority.datasourceSpec.getId());
+    }
```

Comment:
this means that from 

```
get-localauthority-data-for-traffic-counters.json, i could remove

"importerClass": "uk.org.tombolo.importer.ons.OaImporter",
 "datasourceId": "localAuthority"
```
but wont this restrict user to compute on localAuthority only and not with msoa or lsoa?

Reply:
Please see the general comment.